src/domains/transaction/components/TransactionDetail/TransactionSummary/TransactionSummary.tsx

```diff
@@ -50,18 +54,16 @@ export const TransactionSummary = ({
 					</>
 				)}
 
-				{transaction.isValidatorResignation() && (
+				{transaction.isValidatorResignation() && validatorFee > 0 && (
```

Comment:
can you make it show even if fee is 0? similar to how we handle it in arkscan where it shows a tooltip with more details to explain the 0 case as well as one with fees

Reply:
applies to the tx details when opening from the transaction table as well as the full flow during resignation so it's visible everywhere